autoscale-api/src/main/java/com/sequenceiq/periscope/api/model/ActivityStatusToApiActivityStatusConverter.java

```diff
@@ -0,0 +1,26 @@
+package com.sequenceiq.periscope.api.model;
+
+import java.util.Map;
+
+import org.springframework.stereotype.Component;
+
+@Component
```

Comment:
This `@Component` must be injected into the other converter and not initialized in the setter mentioned above